CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 🔥 *Features*
 * Add .project property to WorkflowRun to get the info about workspace and project of running workflow
+* VersionMismatch warnings are shown only when interacting with specific workflow runs, not while listing workflows.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏Some formatting is nice in the markdown.
```suggestion
* `VersionMismatch` warnings are shown only when interacting with specific workflow runs, not while listing workflows.
* ```